tasks/fix-cat2.yml

```diff
@@ -940,12 +940,21 @@
       - notimplemented
 
 - name: "MEDIUM | RHEL-07-021030 | PATCH | All world-writable directories must be group-owned by root, sys, bin, or an application group."
-  command: "true"
-  changed_when: no
+  block:
+    - name: "MEDIUM | RHEL-07-021030 | AUDIT | All world-writable directories must be group-owned by root, sys, bin, or an application group."
+      command: find {{ rhel7stig_local_mounts | join(' ') }} -type d -perm -002 -gid +999
```

Comment:
Will want `-xdev`

Reply:
My apologies for the delay. Will get back into this and resolve it today if at all possible.